cves/CVE-2017-14056.yml

```diff
@@ -186,8 +201,11 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    It is possible that it could have been found by a complex fuzzer,
+    as a reasonable exploit to blindly try would be a file with header data 
```

Comment:
This part, `as a resonable exploit to blind try would be...` reads a bit oddly to me, I'd restate this.

I do agree that a complex fuzzer could have found this 👍